Shared/Samples/Render multilayer symbols/RenderMultilayerSymbolsView.swift

```diff
@@ -0,0 +1,453 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct RenderMultilayerSymbolsView: View {
+    /// A map with a light gray basemap.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISLightGray)
+        map.initialViewpoint = Viewpoint(boundingGeometry: Point(x: -713_800, y: 0, spatialReference: .webMercator))
+        return map
+    }()
+    
+    /// The graphics overlay containing the multilayer graphics and associated text graphics.
+    @State private var graphicsOverlay = GraphicsOverlay()
+    
+    /// The offset used to keep a consistent distance between symbols in the same column.
+    private let offset = 20.0
```

Comment:
Maybe this should be `offsetBetweenSymbols`? `offset` is a little too broad and could refer to many things.